packages/shared/graphql/generated/GetTestPreviewsForWorkspace.ts

```diff
@@ -0,0 +1,55 @@
+/* tslint:disable */
+/* eslint-disable */
+// @generated
+// This file was automatically generated and should not be edited.
+
+// ====================================================
+// GraphQL query operation: GetTestPreviewsForWorkspace
+// ====================================================
+
+export interface GetTestPreviewsForWorkspace_node_Recording {
+  __typename: "Recording";
+}
+
+export interface GetTestPreviewsForWorkspace_node_Workspace_tests_edges_node_stats {
```

Comment:
I think GraphQL queries, like method names, should have semantic meaning.

What does a "test preview" mean in this case? You're loading these into a thing called a `testCache` which seems like it should contain "tests" but then you have a different GraphQL query called `GetTest...` so what is a "test preview" and why does that go into a "test cache"?

I think the names we choose for this stuff are important because other people will have to understand and maintain this code.

Reply:
I renamed this to `GetWorkspaceTests`

Reply:
I appreciate that but I'm not sure it's any clearer. Now we have `GetTestForWorkspace` and `GetWorkspaceTests`.